src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -89,7 +55,7 @@ public WebElement forElementVisible(WebElement element) {
     try {
       element.getTagName();
     } catch (WebDriverException e) {
-      PageObjectLogging.log(INIT_MESSAGE, INIT_ERROR_MESSAGE, true);
+      PageObjectLogging.log("INIT ELEMENT", "PROBLEM WITH ELEMENT INIT", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "PROBLEM WITH ELEMENT INIT" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "INIT ELEMENT" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)